src/foundry/client/data/documents/user.d.mts

```diff
@@ -652,7 +654,7 @@ declare global {
      *
      * @param permission - The permission name from USER_PERMISSIONS
      * @param allowed    - Whether to allow or restrict the permission
-     * @throws If the calling user is not at least an Assistant GM
+     * @remarks @throws If the calling user is not at least an Assistant GM
```

Comment:
I thought we decided on two lines? Me checking Discord's rendering right after was probably confusing.